docs/_includes/header.html

```diff
@@ -1,4 +1,4 @@
-<header class="s-topbar stacks-topbar ps-fixed h64 js-stacks-topbar print:d-none">
+<header class="s-topbar stacks-topbar ps-fixed js-stacks-topbar print:d-none">
```

Comment:
Is this intentional? It doesn't seem related to this PR. Maybe it could be in its separate commit.

Reply:
Good catch! This is a remnant from some premature optimization to support variable spacing. I have some local changes that I neglected to commit that support some spacing theming in the docs, but this change slipped by me. I'll remove it.